tools/project_manager/autorelease.inc

```diff
@@ -563,33 +566,37 @@ function AP_setup( $round )
 
 function AP_teardown( $round )
 {
-    mysqli_query(DPDatabase::get_connection(), "
-        DROP TABLE active_page_counts
-    ") or die(DPDatabase::log_error());
+    $sql = "DROP TABLE active_page_counts";
+    DPDatabase::query($sql);
 }
 
 function AP_add_project( $round, $projectid )
 {
-    mysqli_query(DPDatabase::get_connection(), "
+    $sql = sprintf("
         INSERT INTO active_page_counts
         SELECT
-            '$projectid',
-            SUM( state != '{$round->page_save_state}' ) as pages
-        FROM $projectid
-    ") or die(DPDatabase::log_error());
+            '%s',
+            SUM( state != '%s' ) as pages
+        FROM %s",
```

Comment:
Test for valid projectid rather than escaping it here.

Reply:
Just insert `$projectid` here directly since it has been validated and is the table name.